README.md

```diff
@@ -1,19 +1,21 @@
 # MediaJel Engineering Coding Challenge
 
+<img src="https://d1ww29wzvuczsv.cloudfront.net/i?&e=pv&page=MediajelCodingChallenge&url=https%3A%2F%2Fgithub.com%2FMediaJel%2FCodingChallenge&aid=mj-code-test&p=web&tv=no-js-0.1.0" />
 
 ## Context
 
-Welcome to Carpe Vinum -  the open source tool for wine lovers to connect, track their tasting sessions, and discover new wine. This tool is used to test potential candidates for our engineering team.
+Welcome to Carpe Vinum - the open source tool for wine lovers to connect, track their tasting sessions, and discover new wine. This tool is used to test potential candidates for our engineering team.
 
 ## Back End Challenge
 
 At the time this test is written, the datamodel associated with this application is partially developed. However, a wine app without wines in the database is not much of a wine app! This challenge will focus on backend development with regards to efficient data processing and modeling. There is also the opportunity to show some flair by using interesting datastore solutions to solve the problem at hand. Before reading the rest of the challenge, it may be benefitial to jump to the "Setup" section to begin getting familiar with the development environment.
 
-Now, for the actual challenge. Assume there's exists a service that can callback in the form of a webhook when new data becomes available. These webhooks will callback with an image of a wine bottle/label. As such, we must gather data from the image to add new content to the database. For the purposes of this challenge, the candidate can use whatever images they choose. We recommend those from the [LCBO](http://www.lcbo.com/content/lcbo/en.html#.XEpE6M9KjUY) website. In any case, tests can be written with mock images to verify the correctness of the solution. 
+Now, for the actual challenge. Assume there's exists a service that can callback in the form of a webhook when new data becomes available. These webhooks will callback with an image of a wine bottle/label. As such, we must gather data from the image to add new content to the database. For the purposes of this challenge, the candidate can use whatever images they choose. We recommend those from the [LCBO](http://www.lcbo.com/content/lcbo/en.html#.XEpE6M9KjUY) website. In any case, tests can be written with mock images to verify the correctness of the solution.
 
```

Comment:
When we discussed together, you mentioned that there was some progress made to the back end? For example - a rest endpoint was set up to access the images wasn't it and Tesaract was installed. Can we give some instructions here to explain how to move this work forward?

Reply:
I forgot about that. I will add it as another task for the challenge :)

Reply:
I would also do a quick QA while you have it up and running to check for any other major bugs. If any are found just mention them in the README.